runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerLog.java

```diff
@@ -61,36 +80,61 @@
 		logTitle.add(titleLabel, BorderLayout.WEST);
 
 		// If we have subtitle, add it
-		if (!Strings.isNullOrEmpty(subTitle))
+		if (!Strings.isNullOrEmpty(entry.getSubTitle()))
 		{
-			final JLabel subTitleLabel = new JLabel(subTitle);
+			subTitleLabel.setText(entry.getSubTitle());
 			subTitleLabel.setFont(FontManager.getRunescapeSmallFont());
 			subTitleLabel.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
 			logTitle.add(subTitleLabel, BorderLayout.CENTER);
 		}
 
-		totalPrice = calculatePrice(items);
+		priceLabel.setFont(FontManager.getRunescapeSmallFont());
+		priceLabel.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
+		logTitle.add(priceLabel, BorderLayout.EAST);
+
+		addEntry(entry);
+
+		add(logTitle, BorderLayout.NORTH);
+		add(itemContainer, BorderLayout.CENTER);
+	}
+
+	void addEntry(LootTrackerEntry entry)
+	{
+		entries.add(entry);
+		totalKills++;
+		totalPrice = calculatePrice();
+
+		rebuildItems();
 
 		if (totalPrice > 0)
 		{
-			final JLabel priceLabel = new JLabel(StackFormatter.quantityToStackSize(totalPrice) + " gp");
-			priceLabel.setFont(FontManager.getRunescapeSmallFont());
-			priceLabel.setForeground(ColorScheme.LIGHT_GRAY_COLOR);
-			logTitle.add(priceLabel, BorderLayout.EAST);
+			priceLabel.setText(StackFormatter.quantityToStackSize(totalPrice) + " gp");
 		}
 
+		if (totalKills > 1)
```

Comment:
same here, make it x 1

Reply:
Then single logs would not show combat, even if grouping was disabled.